docs/arcaflow/getting-started.md

```diff
@@ -105,142 +189,488 @@ input:
+        "const": "success"
+      },
+      "output_data": {
+        "type": "object",
+        "properties": {
+          "message": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "message"
+        ],
+        "additionalProperties": false,
+        "dependentRequired": {}
+      }
+    },
+    {
+      "output_id": {
+        "type": "string",
+        "const": "error"
+      },
+      "output_data": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "error"
+        ],
+        "additionalProperties": false,
+        "dependentRequired": {}
+      }
+    }
+  ],
+  "$defs": {
+    "SuccessOutput": {
+      "type": "object",
+      "properties": {
+        "message": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "message"
+      ],
+      "additionalProperties": false,
+      "dependentRequired": {}
+    },
+    "ErrorOutput": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "error"
+      ],
+      "additionalProperties": false,
+      "dependentRequired": {}
+    }
+  }
+}
+```
 
-## Step 7: Run the workflow
+A plugin takes its input as a file, but because it runs as a container, it looks for the input file in the context of the container. This means you either need to bind-mount the input file to the container, or, as in this example, pipe the input value to the plugin's file input.
 
-Finally, let's run our workflow. Make sure you are in the directory where the workflow is located.
+```yaml title="input.yaml"
+name:
+  _type: nickname
+  nick: Arcalot
+```
 
-=== "Linux/MacOS"
+!!! note
+    In order to pipe the input to the container, you must pass the `-i, --interactive` parameter.
 
+=== "Podman"
+    ```bash
+    cat input.yaml | podman run -i --rm quay.io/arcalot/arcaflow-plugin-example -f -
     ```
-    /path/to/arcaflow -input input.yaml -config config.yaml
+=== "Docker"
+    ```bash
+    cat input.yaml | docker run -i --rm quay.io/arcalot/arcaflow-plugin-example -f -
     ```
 
-=== "Windows"
+```yaml title="example plugin return YAML"
+output_id: success
+output_data:
+  message: Hello, Arcalot!
+debug_logs: ''
+```
+
+Now let's generate a UUID with the utilities plugin. This plugin has multiple steps, so we need to specify which step to run. The step also requires no input, so we pass it an empty object.
+
+!!! note
+    An input object is always required, even if a plugin step does not require input parameters.
 
+=== "Podman"
+    ```bash
+    echo '{}' | podman run -i --rm quay.io/arcalot/arcaflow-plugin-utilities -s uuid -f -
     ```
-    C:\path\to\arcaflow.exe -input input.yaml -config config.yaml
+=== "Docker"
+    ```bash
+    echo '{}' | docker run -i --rm quay.io/arcalot/arcaflow-plugin-utilities -s uuid -f -
     ```
```

Comment:
Using `echo` into a pipe is almost as dismaying as using `cat`.  Allow me to recommend a "Here String":
```bash
podman run -i --rm quay.io/arcalot/arcaflow-plugin-utilities -s uuid -f - <<<'{}'
```

Ditto at line 651.

Reply:
Or, of course, `<<<'{}' podman run -i --rm [...]`

Reply:
I disagree. As with the `cat` instance above, I think the `echo` method here is more approachable and much more clear, even though it isn't the most technically astute method.